var/spack/repos/builtin/packages/py-maestrowf/package.py

```diff
@@ -38,6 +41,7 @@ class PyMaestrowf(PythonPackage):
 
     depends_on('python@2.7:2.8,3.5:',  type=('build', 'run'))
     depends_on('py-setuptools',        type='build')
+    depends_on('py-poetry',            type=('build'),        when='@1.1.9:')
```

Comment:
Would:
```suggestion
    depends_on('py-poetry-core',       type=('build'),        when='@1.1.9:')
```
be fine too?

Reply:
I don't expect this to cause issues, can test this locally.

Reply:
The point is that to build it's very likely you don't need `py-poetry`, just the `core` (which brings in a lot less dependencies).

Reply:
Unfortunately this won't work. The `build-backend` is listed as `poetry.mason.api`, meaning that the full `py-poetry` suite is required. If the developers change the `build-backend` in the future then `py-poetry-core` would likely be sufficient.

Reply:
Yep, that makes sense. I expect it will work, just doing a sanity check.

Reply:
I'm actually sating my own curiosity, so testing it locally with `py-poetry-core`

Reply:
@adamjstewart -- you're correct. I'll revert this back to `py-poetry`. I'm not too concerned about the extra dependencies at the moment, eventually this will be released as a non-development version so it would be coming from a wheel anyway. Is that a concern on your end?

Reply:
No concerns on my end. All poetry deps are quick to build. It seems like the community is moving towards using poetry-core unless absolutely necessary so something to think about for future releases.